tests/e2e/common.go

```diff
@@ -21,12 +21,13 @@ import (
 )
 
 var (
-	dynamicClient dynamic.DynamicClient
-	testEnv       *envtest.Environment
-	ctx           context.Context
-	cancel        context.CancelFunc
-	suiteTimeout  = 5 * time.Minute
-	testTimeout   = 2 * time.Minute
+	dynamicClient       dynamic.DynamicClient
+	testEnv             *envtest.Environment
+	ctx                 context.Context
+	cancel              context.CancelFunc
+	suiteTimeout        = 30 * time.Minute
+	testTimeout         = 2 * time.Minute
+	testPollingInterval = 1 * time.Second
```

Comment:
I think it will be better to poll the K8S API once per second rather than the default once per 10ms just in case we ever do face a rate limiting issue